common/flatpak-dir.c

```diff
@@ -1839,6 +1841,154 @@ get_system_locations (GCancellable *cancellable,
   return g_steal_pointer (&locations);
 }
 
+char **
+flatpak_get_preinstall_config_file_paths (GCancellable *cancellable,
+                                          GError      **error)
+{
+  g_autoptr(GPtrArray) paths = NULL;
+  g_autoptr(GFile) conf_dir = NULL;
+  g_autoptr(GFileEnumerator) dir_enum = NULL;
+  g_autoptr(GError) my_error = NULL;
```

Comment:
```suggestion
  g_autoptr(GError) local_error = NULL;
```

Reply:
I'd prefer local_error as well, but the rest of the flatpak-dir.c file seems to use my_error ... Let's keep this the same for now and I can do a followup afterwards and try to update the whole file to use local_error, if maintainers agree.